src/components/TopBar.tsx

```diff
@@ -252,7 +249,28 @@ export const TopBar = (props: TopBarProps): ReactNode => {
                 </Clickable>
               </div>
             )}
-            {!compact && (
+            {isScientificServices() && (
+              <>
+                <NavSection
+                  href='https://broadscientificservices.zendesk.com/hc/en-us/categories/39899029682331'
```

Comment:
q: is this the right link to our release notes?

Reply:
i think technically https://broadscientificservices.zendesk.com/hc/en-us/sections/40014227087643-Release-notes is where the release notes will live but this link can be found in your link so either works.